astropy/time/core.py

```diff
@@ -1871,11 +1834,26 @@ def sidereal_time(self, kind, longitude=None, model=None):
 
         Returns
         -------
-        sidereal time : `~astropy.coordinates.Longitude`
-            Sidereal time as a quantity with units of hourangle
-        """  # docstring is formatted below
+        `~astropy.coordinates.Longitude`
+            Local sidereal time, with units of hourangle.
 
-        from astropy.coordinates import Longitude
+        See Also
+        --------
+        astropy.time.Time.earth_rotation_angle
+
+        Notes
+        -----
+        The difference between apparent sidereal time and Earth rotation angle
+        is the equation of the origins which is the angle between the equinox
+        and the CIO. Applying apparent sidereal time to the hour angle yields
```

Comment:
The equation of the origins is actually defined as the angle between the CIO and the equinox. I know that I got that backwards at first, but I did edit my original comment to correct that. Sorry for the confusion on my part.

Reply:
No worries; I copied it before the change (perhaps from the e-mail I got, not sure any more...)